pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -639,6 +748,108 @@ const New = () => {
                 }}
               ></VoteBySwitch>
             )}
+            <div className="max-w-lg w-full mb-4 flex flex-wrap justify-between items-end">
+              <ProposalTypeRadioButton
+                onClick={() => setIsMulti(false)}
+                selected={!isMulti}
+                disabled={false}
+                className="w-60"
+              >
+                Executable
+              </ProposalTypeRadioButton>
+              <div className="flex flex-col items-center justify-evenly">
+                <div className="bg-[#10B981] text-black flex flex-row gap-2 text-sm 
+                items-center px-2 py-1 rounded-md mb-2 w-60">
+                  <TableOfContents />
+                  <div>New: Multiple Choice Polls</div>
+                </div>
+                <ProposalTypeRadioButton
+                  onClick={() => setIsMulti(true)}
+                  selected={isMulti}
+                  disabled={false}
+                  className="w-60"
+                >
+                  Non-Executable <br /> (Multiple-Choice)
+                </ProposalTypeRadioButton>
+              </div>
+            </div>
+            {isMulti ?
+              <div className="mt-8 mb-8">
```

Comment:
let's just move all the multichoice form stuff to its own component, this component is getting way too big

Reply:
you can move the single choice vote form to its own component if you want, or not bother. but at least the new stuff should just be its own component.

Reply:
Done, new component `MultiChoiceForm` is created